java/src/main/java/com/saucelabs/saucebindings/SauceOptions.java

```diff
@@ -128,6 +128,12 @@
         knownCITools.put("TeamCity", "TEAMCITY_PROJECT_NAME");
     }
 
+    public static SauceOptions headless() {
```

Comment:
Nice and easy :) Does it need to be `static`?

Reply:
to return an instance, yes, that's what the static methods are for.